### ISSUE TITLE
Travis CI doesn't support Python 3.x under OS X 

### ISSUE THREAD

**Issue by osvenskan** [osvenskan/posix_ipc#12]
I'd like to add support for testing under OS X but Travis doesn't support Python under that OS yet.

**Comment by osvenskan** [comment]
It's now supported: https://docs.travis-ci.com/user/reference/osx/

**Comment by osvenskan** [comment]
This is improved with #20 which now gets Travis to test under macOS, but only with Python 2.7. I tried this `travis.yml` syntax to get it to run under Python 3, but it failed with the promising error that `posix_ipc` wasn't installed --
```
matrix:
  include:
    - { python: 3.7, dist: xenial, sudo: true }
    - os: osx
      osx_image: xcode10.2  # Python 2.7 & 3.7
      language: shell       # 'language: python' is an error on Travis CI macOS
      script: python3 -m unittest discover
```

So I added an `install` line, but that only got me back to testing under Python 2.7 --
```
matrix:
  include:
    - { python: 3.7, dist: xenial, sudo: true }
    - os: osx
      osx_image: xcode10.2  # Python 2.7 & 3.7
      language: shell       # 'language: python' is an error on Travis CI macOS
      install: python3 setup.py install
      script: python3 -m unittest discover
```

For now I'm going to content myself with changing the ticket title to make it Python 3.7-specific and hope that Travis support improves for Python on the Mac.

**Comment by osvenskan** [comment]
fixed by PR #23/version 1.0.5
